### PR TITLE
Add Getting Started instructions

### PR DESCRIPTION
**Pull request by RobLoach**
This adds some getting started instructions to the README

References #7

### CODE REVIEW

**Comment by RobLoach** [comment]
@MirrasHue @sbgssol This is pretty slim, so I'd love some thoughts on how it could improve.

### Getting Started

*raylib-cpp* is a header-only library. This means in order to use it, you must link your project to [raylib](https://www.raylib.com/), and then include [`raylib/raylib.hpp`](raylib-cpp/include/raylib/raylib.hpp).

1. Set up a *raylib* project using the [build and install instructions](https://github.com/raysan5/raylib#build-and-installation)
2. Download *raylib-cpp*
3. Include [`raylib/raylib.hpp`](raylib-cpp/include/raylib/raylib.hpp)
    ``` cpp
    #include "raylib/raylib.hpp"
    ```

Should probably link to the [example projects](projects) too... Adding the VSCode examples would be mucho appreciated if you have them.

File: README.md [hunk 1]
```diff
@@ -157,8 +159,25 @@ cat
 	.Resize(150, 200);
 ```
 
+### Getting Started
```
Comment (RobLoach) [inline README.md:162]:
``` suggestion
## Getting Started
```

File: README.md [hunk 2]
```diff
@@ -157,8 +159,25 @@ cat
...
+*raylib-cpp* is a header-only library. This means in order to use it, you must link your project to [raylib](https://www.raylib.com/), and then include [`raylib/raylib.hpp`](raylib-cpp/include/raylib/raylib.hpp).
+
+1. Set up a *raylib* project using the [build and install instructions](https://github.com/raysan5/raylib#build-and-installation)
+2. Download *raylib-cpp*
+3. Include [`raylib/raylib.hpp`](raylib-cpp/include/raylib/raylib.hpp)
+    ``` cpp
+    #include "raylib/raylib.hpp"
+    ```
+
+#### Starter Projects
```
Comment (RobLoach) [inline README.md:173]:
``` suggestion
### Starter Projects
```

**Comment by MirrasHue** [comment]
@RobLoach, I managed to get raylib working on CodeBlocks with this tutorial: [Raylibs setup on Codeblocks](https://www.youtube.com/watch?v=a2IUxJFr8sg), the template project from raylib is outdated and doesn't work.
After this I tested out some of your examples, and [`core_world_screen.cpp`](https://github.com/RobLoach/raylib-cpp/blob/master/examples/core/core_world_screen.cpp) isn't quite working, the cube doesn't show up, I also tried the equivalent example from raylib's page and that run just nice.

**Comment by MirrasHue** [comment]
Any ideia of what this could be?

**Pull request merged**